src/bootstrap/src/core/config/config.rs

```diff
@@ -1141,6 +1142,7 @@ define_config! {
         download_rustc: Option<StringOrBool> = "download-rustc",
         lto: Option<String> = "lto",
         validate_mir_opts: Option<u32> = "validate-mir-opts",
+        std_features: Option<Vec<String>> = "std-features",
```

Comment:
```suggestion
        std_features: Option<BTreeSet<String>> = "std-features",
```

Reply:
oh got it, I haven’t read into the details a lot - but would the parser parse the BTreeSet directly from a slice in the toml? I assumed that a `Vec<String>` would be straightforward, but let me try this.